src/Views/Blog/Post.cshtml

```diff
@@ -39,7 +39,7 @@
         }
 
         <a href="@Model.GetLink()#comments" itemprop="discussionUrl" title="Go to the comments section">
-            <span itemprop="commentCount">@Model.Comments.Count</span> comments
```

Comment:
Should be `Model.Comments.Count > 1 ? "comments" : "comment"` otherwise it will output "0 comments" if not any.